src/template.njk

```diff
@@ -18,10 +18,11 @@
       <link rel="apple-touch-icon" href="{{ assetPath | default('/assets') }}/images/govuk-apple-touch-icon.png">
     {% endblock %}
 
-    {% block head %}{% endblock %}
-
+    {% block head %}
```

Comment:
This is a breaking change since before even when you set head you'd get the default open graph image.

Could this be solved by something like

```html
<meta property="og:image" content="{{ opengraphAssetPath | default('') }}{{ assetPath | default('/assets') }}/images/govuk-opengraph-image.png">
```

Reply:
tru dat. updated so that the variable can just be set